staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2456,6 +2456,11 @@ type MigrationConfiguration struct {
 	// Network is the name of the CNI network to use for live migrations. By default, migrations go
 	// through the pod network.
 	Network *string `json:"network,omitempty"`
+	// By default, the SELinux level of target virt-launcher pods is forced to the level of the source virt-launcher.
+	// When set to true, DisableSELinuxLevelOnTarget lets the CRI auto-assign a random level to the target.
+	// That will ensure the target virt-launcher doesn't share categories with another pod on the node.
+	// However, migrations will fail when using RWX storage that doesn't automatically deal with SELinux levels.
+	DisableSELinuxLevelOnTarget *bool `json:"disableSELinuxLevelOnTarget,omitempty"`
```

Comment:
Naming is hard. Just suggestion `MatchSELinuxLevelOnMigration`

Reply:
Yeah that was my original idea, but I don't really like boolean options that are true by default, so I tried to flip it around.
However, I agree with you that it doesn't look great, will change.